client/layout/masterbar/style.scss

```diff
@@ -880,6 +879,9 @@ a.masterbar__quick-language-switcher {
 	align-items: center;
 	margin-left: 24px;
 
+	@include breakpoint-deprecated( "<480px" ) {
```

Comment:
Probably not important since I see that this file uses breakpoints in every kind of way, but a good strategy is to pick "desktop first" or "mobile first" and write your CSS that way. This breakpoint is desktop first (the breakpoint actives some rules only when things get too thin) and I usually lean toward mobile first. I notice the same approach for the breakpoints added below, and I'm fine with that, but I also see that some of the checkout masterbar breakpoints are the other way around. 🤷 I'm not sure what to recommend here but I wanted to point it out.